include/api.php

```diff
@@ -743,13 +743,27 @@ function api_get_user(App $a, $contact_id = null)
 
 	$pcontact_id  = Contact::getIdForURL($uinfo[0]['url'], 0, true);
 
+	if (!empty($profile[0]['pdesc'])) {
+		$description = $profile[0]['pdesc'];
+	} else {
+		$description = $uinfo[0]["about"];
+	}
+
+	if (!empty($usr[0]['default-location'])) {
+		$location = $usr[0]['default-location'];
+	} elseif (!empty($uinfo[0]["location"])) {
+		$location = $uinfo[0]["location"];
+	} else {
+		$location = network_name;
```

Comment:
There’s a missing dollar sign in front of “network_name”

Reply:
Done